src/biotite/structure/io/pdbx/convert.py

```diff
@@ -1481,6 +1481,7 @@ def get_assembly(
     extra_fields=None,
     use_author_fields=True,
     include_bonds=False,
+    include_sym_id=False,
```

Comment:
I think we do not need this additional parameter: The extra annotation has only a small overhead, so I think it is OK to always include the `sym_id`.